src/overlays/actors/ovl_Object_Kankyo/z_object_kankyo.c

```diff
@@ -32,41 +57,879 @@ const ActorInit Object_Kankyo_InitVars = {
+                    dz = globalCtx->view.lookAt.z - globalCtx->view.eye.z;
+                    dist = sqrtf(SQ(dx) + SQ(dy) + SQ(dz));
+
+                    baseX = globalCtx->view.eye.x + dx / dist * 80.0f;
+                    baseY = globalCtx->view.eye.y + dy / dist * 80.0f;
+                    baseZ = globalCtx->view.eye.z + dz / dist * 80.0f;
+
+                    this->effects[i].dirPhase.x += 0.049999997f * Rand_ZeroOne();
+                    this->effects[i].dirPhase.z += 0.049999997f * Rand_ZeroOne();
+                    this->effects[i].pos.x += sinf(this->effects[i].dirPhase.x * 0.01f);
+                    this->effects[i].pos.z += cosf(this->effects[i].dirPhase.z * 0.01f);
+                    this->effects[i].pos.y += -this->effects[i].speed;
+
+                    if (this->effects[i].base.y + this->effects[i].pos.y < this->actor.world.pos.y ||
+                        this->effects[i].base.y + this->effects[i].pos.y < globalCtx->view.eye.y - 150.0f) {
+                        this->effects[i].state++;
+                    }
+
+                    maxDist = 80;
+                    if (this->effects[i].base.x + this->effects[i].pos.x - baseX >  maxDist ||
+                        this->effects[i].base.x + this->effects[i].pos.x - baseX < -maxDist ||
+                        this->effects[i].base.y + this->effects[i].pos.y - baseY >  maxDist ||
+                        this->effects[i].base.y + this->effects[i].pos.y - baseY < -maxDist ||
+                        this->effects[i].base.z + this->effects[i].pos.z - baseZ >  maxDist ||
+                        this->effects[i].base.z + this->effects[i].pos.z - baseZ < -maxDist) {
+
+                        // when off screen, wrap around to the other side
+                        if (this->effects[i].base.x + this->effects[i].pos.x - baseX > maxDist) {
+                            this->effects[i].base.x = baseX - maxDist;
+                            this->effects[i].pos.x = 0.0f;
+                        }
+                        if (this->effects[i].base.x + this->effects[i].pos.x - baseX < -maxDist) {
+                            this->effects[i].base.x = baseX + maxDist;
+                            this->effects[i].pos.x = 0.0f;
+                        }
+                        if (this->effects[i].base.z + this->effects[i].pos.z - baseZ > maxDist) {
+                            this->effects[i].base.z = baseZ - maxDist;
+                            this->effects[i].pos.z = 0.0f;
+                        }
+                        if (this->effects[i].base.z + this->effects[i].pos.z - baseZ < -maxDist) {
+                            this->effects[i].base.z = baseZ + maxDist;
+                            this->effects[i].pos.z = 0.0f;
+                        }
+                    }
+                    break;
+
+                case 2:
+                    this->effects[i].state = 0;
+                    break;
+            }
+
+            if (1) {} if (1) {}
```

Comment:
the formatter should be run on this file

Reply:
it was, but some of the changes it made are painful
for example, it's much easier for me to read this
```c
Gfx_TwoTexScroll(globalCtx->state.gfxCtx,
                 0, globalCtx->state.frames * 5, globalCtx->state.frames * 10, 32, 64,
                 1, globalCtx->state.frames * 5, globalCtx->state.frames * 10, 32, 64));
```
than what clang format turns it into.
Also the way it mutilates definitions :/
but I can rerun it I guess

Reply:
yeah some of its changes are unfortunate. but we cant really pick and choose in edge cases

at some point we may want to explore different formatting configurations or even a different formatter, but for now we have to be consistent about using it